ethportal-api/src/discv5.rs

```diff
@@ -1,7 +1,7 @@
+use crate::types::discv5::{NodeInfo, RoutingTableInfo};
+use crate::types::enr::Enr;
+use crate::types::node_id::NodeId;
```

Comment:
This brings up the question of import order formatting... technically it's defined in the style guide to follow the style we used back in python land.. but we aren't that consistent across our codebase. It's definitely not a blocker for this pr, but maybe something we should discuss in our next sync to reach upon an agreed upon scheme.

Reply:
Ah, I didn't do anything special here, just ran `cargo fmt --all`, which did reordering as it saw fit. What were you expecting here?

Reply:
AFAI, `cargo fmt --all` alphabetically orders every block of imports. So this is what I would expect to see. However, if we were "strictly" adhering to the style guide, I'd expect to see `jsonrpsee::...` on line 1 (since it's an external crate), then an empty line, then this block of `crate::...` (i.e. internal) imports. But, this style guide hasn't been enforced consistently across the codebase. This pr just reminded me that maybe it's something we want to discuss, or maybe it's not much of a problem at all